app/components/layer-advanced/template.hbs

```diff
@@ -1,5 +1,12 @@
 {{edit-text showSettings=showSettings editMode=editMode layer=layer type="title"}}
 
-<div id="froala-editor"  class="fr-view">
-</div>
+<!-- <div id="froala-editor"  class="fr-view">
+</div> -->
 
+{{#if editMode}}
+	{{quill-editor value=layer.content options=options textChange=(action "updateText")}}
```

Comment:
I'm going to leave the border color for now because I might be implementing the bubble theme later on after I experiment with it some more. it is the inline version of the editor but I need to test a few things before i consider it, will be  a future PR